backend/static/styles/styles.css

```diff
@@ -312,3 +312,8 @@ and also iPads specifically.
       background-color: #eee; }
     .watch-table.pure-table-striped tr:nth-child(2n-1) td {
       background-color: inherit; } }
+
+/* Token table toggle on settings page */
+#token-table {
```

Comment:
Sure thing, have you considered using a tool like webpack? Could automate it in the Dockerfile really easily.

Reply:
done 
f1f6690

Reply:
webpack - unsure, i always feel that these style of tools make it almost impossible for anyone not uber front-end nerdy to commit anything, i wasnt even sure about using SASS but it started to get too messy :) I'm still interested in seeing what you'de do with with webpack, so could be nice to see that on another branch/PR for sure (Disclaimer: I dont fully understand tools like webpack)

Reply:
Fair point about keeping it simple. Once you start using webpage in a project it’s impossible to get off it haha.

Have you heard of [fantastic](http://www.fanstatic.org/en/latest/)? I haven’t used it before, but I feel like it could fit really well. You just declare your sass files in the setup and just let Python handle everything. Even for JavaScript. It will grab the source js/sass, compile/minifiy etc and insert it into your pages all for you. You could dump npm/node altogether this way and keep it all in Python! 

An example on [this](http://www.fanstatic.org/en/latest/compilers.html) page using it on sass.